### DISCUSSION TITLE
I cant use my domain for my project

### DISCUSSION THREAD

**Discussion by KwaliOfficial** [vercel/community#4992] [Help]
### Summary

Im trying to use the domain i purchased from another provider by changing the dns to vercel, but the problem is that our programmer who has another account tried to use our domain on his account and even though he deleted the domain from his account, it still wont let us use the domain in our account , it says that another user is using it and is pending verification. Its been like this for 4 days now. 

### Steps to Reproduce

Under  the domain section it shows this text: "Another Vercel account is using this domain. Please set the following TXT record..."

**Comment by amyegan** [comment]
Hi @KwaliOfficial. Have you already added the required TXT record?

**Reply by KwaliOfficial** [reply]
where do i have to add it? on my provider's site?

**Reply by amyegan** [reply]
Yes, the TXT record should be added with your domain registrar 🙂
